src/extensions/default/bramble/main.js

```diff
@@ -352,16 +291,15 @@ define(function (require, exports, module) {
             // Set initial theme
             Theme.setTheme(data.theme);
 
-            window.removeEventListener("message", _getInitialDocument);
-
-            window.addEventListener("message", _buttonListener);
+            window.removeEventListener("message", _getInitialDocument, false);
+            window.addEventListener("message", RemoteCommandHandler.handleRequest, false);
 
             var fileHTML = FileSystem.getFileForPath("/index.html");
             var fileCSS  = FileSystem.getFileForPath("/style.css");
             var fileJS   = FileSystem.getFileForPath("/script.js");
 
             // Write the HTML file and block on it being done.
-            fileHTML.write(data.source ? data.source : defaultHTML,
+            fileHTML.write(data.source ? data.source : defaultHTML, {blind: true},
```

Comment:
Yeah, without it you run into issues where it tries to compare against an in memory cache.  See https://github.com/adobe/brackets/blob/master/src/file/FileUtils.js#L96-L108.  It just forces our write to work, despite what's on the filesystem.